tamr_toolbox/workflow/concurrent/PlanNodeStatus.py

```diff
@@ -0,0 +1,75 @@
+"""Enum representing job status"""
+from enum import IntEnum
+import logging
+from tamr_unify_client.operation import Operation
+
+import tamr_toolbox
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PlanNodeStatus(IntEnum):
+    """
+    A class representing job status for executing a concurrent pipeline
+    """
+
+    FAILED = -2  # one or more of the jobs for the project failed
+    CANCELLED = -1  # one or more of the jobs for the project was cancelled
+    BLOCKED = 0  # running this project is blocked by upstream dependencies failing
+    PLANNED = 1  # default state - planned but not yet runnable due to upstream dependencies
+    SKIPPABLE = 2  # used when, for example, starting later in the plan
+    RUNNABLE = 3  # all dependencies met (e.g. starting tier project) - the project can be run
+    PENDING_NEXT_STEP = 4  # project has run some steps successfully and has more steps remaining
+    RUNNING = 5  # a job for the project is currently running
+    SUCCEEDED = 6  # all project steps completed successfully
+
+
+def from_tamr_op(op: Operation) -> PlanNodeStatus:
+    LOGGER.debug(f"checking state of operation {op}")
+    # check each operation
+    if op.state == "PENDING" or op.state == "RUNNING":
+        return PlanNodeStatus.RUNNING
+    elif op.state == "SUCCEEDED":
+        return PlanNodeStatus.SUCCEEDED
+    elif op.state == "FAILED":
+        return PlanNodeStatus.FAILED
+    elif op.state == "CANCELLED":
```

Comment:
it turns out that the op response for Cancelled jobs actually has state: "CANCELED", at least on v2021.006.
```suggestion
    elif op.state == "CANCELLED" or op.state == "CANCELED":
```

Reply:
great catch - this was just a typo

Reply:
apparently it's British vs American spelling